authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/manager/AuthenticatorBridgeManagerImpl.kt

```diff
@@ -103,6 +106,19 @@ internal class AuthenticatorBridgeManagerImpl(
         )
     }
 
+    override fun startAddTotpLoginItemFlow(totpUri: String): Boolean =
+        bridgeService
+            ?.safeCall {
+                // Grab symmetric key data from local storage:
+                val symmetricKey = symmetricKeyStorageProvider.symmetricKey ?: return@safeCall false
+                // Encrypt the given URI:
+                val addTotpData = AddTotpLoginItemData(totpUri).encrypt(symmetricKey).getOrThrow()
+                return@safeCall this.startAddTotpLoginItemFlow(addTotpData)
+            }?.fold(
+                onFailure = { false },
+                onSuccess = { true }
+            ) ?: false
```

Comment:
Wanna update the chaining:

```kotlin
            }
            ?.fold(
                onFailure = { false },
                onSuccess = { true }
            )
            ?: false
```

Reply:
I do! Thank you. 

e615633712a3e2f0ff226cf818af2ecc90d4bbea